CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Next release
+* Add support for `columns` and `additional_columns` on Report creation
+* Re-add `Report.retrieve` function
```

Comment:
The `retrieve` function isn't missing, it gets inherited.

Reply:
Resolved in 838f54d0dbb44f5b78595d4a0840983b7d997834